docs/guide/rest-resources.md

```diff
@@ -244,4 +244,41 @@ will also include the pagination information by the following HTTP headers:
 * `X-Pagination-Per-Page`: The number of resources in each page;
 * `Link`: A set of navigational links allowing client to traverse the resources page by page.
 
+Since collection in REST APIs is a data provider, it shares all data provider features i.e. pagination and sorting.
+
 An example may be found in the [Quick Start](rest-quick-start.md#trying-it-out) section.
+
+### Filtering collections <span id="filtering-collections"></span>
+
+Since version 2.0.13 Yii provides a facility to filter collections. In case you are using `ActiveController` you can enjoy it
+right away. An example can be found in the [Quick Start](rest-quick-start.md#trying-it-out) section. In case you're
+implementing an endpoint yourself, filtering could be done like the following:
+
+```php
+$filter = new ActiveDataFilter([
+    'searchModel' => 'app\models\PostSearch'
```

Comment:
how does this model look like, there is no standard for this as far as I see.

Reply:
It's a regular form model. Just public properties and rules. I'll add an example.

Reply:
in base PR I can see DynamicModel and anonymous function... 
```php
    public function actions()
    {
        return [
            'index' => [
                'class' => 'yii\rest\IndexAction',
                'modelClass' => $this->modelClass,
                'dataFilter' => [
                    'class' => 'yii\rest\ActiveDataFilter',
                    'searchModel' => function () {
                        return (new \yii\base\DynamicModel(['id' => null, 'name' => null]))
                            ->addRule('id', 'integer')
                            ->addRule('name', 'trim')
                            ->addRule('name', 'string');
                    },
                ],
            ],
...
        ];
    }

```
Can process anonymous class `new class()` instead?